bigbluebutton-html5/imports/api/captions/server/handlers/captionUpdate.js

```diff
@@ -1,11 +1,19 @@
-import Captions from '/imports/api/captions';
-import { logger } from '/imports/startup/server/logger';
+import Logger from '/imports/startup/server/logger';
+import { check } from 'meteor/check';
+import addCaption from '../modifiers/addCaption';
 
-export function updateCaptionsCollection(meetingId, locale, payload) {
+const CAPTION_CHUNK_LENGTH = 1000;
```

Comment:
Please pull the value for CAPTION_CHUNK_LENGTH = 1000 from a config file. Now you redefine it in different files